docs/source/getting-started/quickstart.rst

```diff
@@ -97,9 +97,20 @@ Let's apply Laplace noise to a value.
             :end-before: /demo
 
 This is obviously not the easiest way to add noise to a number,
-but it demonstrates a number of OpenDP patterns:
+but it demonstrates a number of low-level OpenDP patterns:
 
 * Defining your metric space with ``space_of`` in Python's Context API, or a (domain, distance) tuple in any language..
 * Chaining operators together with ``>>`` in Python and Rust, or ``|>`` in R.
 * Constructing a ``Measurement`` function on your metric space with ``then_laplace``.
 * Invoking that measurement on a value to get a DP release.
+
+OpenDP has layered APIs which provide increasing abstraction and usability:
+
+* The **Framework API** is low-level. Available for Python and R, it mirrors the underlying Rust framework.
+* The **Context API** introduces a ``Context`` class which ensures that queries do not exceed the privacy budget. Currently available only for Python.
+* The **Polars API** provides a DP extension to the `Polars <https://docs.pola.rs/>`_ dataframe library. Currently available only for Python.
```

Comment:
I think it would be better to just state that we have two APIs: the Context API, and the Framework API. I'd list the Context API first. 

```
* The **Context API** is simpler to use and enforces best practices. The API is designed to ensure that mistakes that lead to under-estimated privacy guarantees cannot be made. Currently available only for Python.
* The **Framework API** is lower-level, and directly follows the OpenDP programming framework. Available for Python, R and Rust.
```

Polars and Sklearn cross-cut these: we aim to make either accessible from either, and we'll probably expand this set. I'd wait to introduce them until the tabular data and statistical modeling sections, respectively.

Reply:
So, essentially restore the original text here, but it still makes sense to move it up to quickstart -- I feel like Polars at least should be mentioned here as well, but maybe it doesn't need to be elevated to a bullet point?

Reply:
That works for me.